shuup/default_reports/forms.py

```diff
@@ -45,6 +48,43 @@ def initial_contacts(self, key):
         return []
 
 
+class OrderLineReportForm(BaseReportForm):
+
+    order_line_type = forms.MultipleChoiceField(
+        label=_("Order Line Type"),
+        required=False,
+        choices=[
+            (line_type.value,
+             line_type.name.capitalize()) for line_type in OrderLineType],
+        initial=[1])  # Because value of OrderLineType.PRODUCT is 1
+
+    def __init__(self, *args, **kwargs):
+        super(OrderLineReportForm, self).__init__(*args, **kwargs)
+
+        supplier = Select2MultipleField(label=_("Suppliers"),
+                                        model=Supplier,
+                                        required=False,
+                                        help_text=_("Filter order lines by suppliers."))
+        order_status = Select2MultipleField(label=_("Order status"),
+                                            required=False,
+                                            model=OrderStatus,
+                                            help_text=_("Filter order lines by status of their order."))
+
+        suppliers = self.initial_supplier("supplier")
+
+        if suppliers:
+            supplier.initial = suppliers
+            supplier.widget.choices = [(obj.pk, obj.name) for obj in suppliers]
+
+        self.fields["supplier"] = supplier
+        self.fields["order_status"] = order_status
+
+    def initial_supplier(self, key):
```

Comment:
Naming of this function indicates that one supplier is return, but instead it returns list or queryset. Maybe `get_initial_suppliers`